app/views/jst/workspace/search/embed_code.jst

```diff
@@ -11,6 +11,9 @@
     <%= options %>
   });
 &lt;/script&gt;
+&lt;noscript&gt;
+  &lt;a href="<%= window.location %>"&gt;View/search document collection&lt;/a&gt;
```

Comment:
Oh, good point re: workspace.

What I was going for was a single link fallback that best approximated what the embedded collection would be showing to a random (and presumably unauthenticated) reader. So yeah, it should be a link to the same query on the public search page. Right?